book/src/requirements.md

```diff
@@ -0,0 +1,67 @@
+# Requirements
+
+This page lists the requirements for running `bindgen` and how to get them.
+
+## Clang
+
+`bindgen` leverages `libclang` to preprocess, parse, and type check C and C++
+header files.
+
+It is recommended to use Clang 3.9 or greater, however `bindgen` can run with
+older Clangs with some features disabled.
+
+* **If you are generating bindings to C,** 3.7 and 3.8 will probably work OK for
+you.
+
+* **If you are generating bindings to C++,** you almost definitely want 3.9 or
+greater.
+
+### Installing Clang 3.9
+
+#### Windows
+
+Download and install the official pre-built binary from
+[LLVM download page](http://releases.llvm.org/download.html).
+
+#### MacOS
```

Comment:
I don't know how much it really matters but Apple's branding uses a lowercase m if you want to follow their guidelines.